src/Context/Cart/MollieResourceBasedCartContext.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+/*
+ * This file has been created by developers from BitBag.
+ * Feel free to contact us once you face any issues or want to start
+ * You can find more information about us on https://bitbag.io and write us
+ * an email on hello@bitbag.io.
+ */
+
+declare(strict_types=1);
+
+namespace BitBag\SyliusMolliePlugin\Context\Cart;
+
+use BitBag\SyliusMolliePlugin\Resolver\Order\MollieResourceOrderResolverInterface;
+use Sylius\Component\Order\Context\CartContextInterface;
+use Sylius\Component\Order\Context\CartNotFoundException;
+use Sylius\Component\Order\Model\OrderInterface;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\RequestStack;
+
+final class MollieResourceBasedCartContext implements CartContextInterface
+{
+    private RequestStack $requestStack;
+
+    private MollieResourceOrderResolverInterface $mollieResourceOrderResolver;
+
+    public function __construct(
+        RequestStack $requestStack,
```

Comment:
This dependency makes the service usable only in HTTP requests. I thought you should know ;)